features/podman/api_daemon.pan

```diff
@@ -0,0 +1,18 @@
+unique template features/podman/api_daemon;
+
+include 'components/systemd/config';
+
+'/software/components/systemd/skip/service' = false;
+
+# This is a customization file
+'/software/components/systemd/unit/podman/file/replace' = false;
+
+prefix '/software/components/systemd/unit/podman/file/config/service';
+'EnvironmentFile' = list('/etc/sysconfig/podman');
+
+'/software/components/systemd/unit' = {
+    SELF['podman'] = nlist('startstop', true);
+    SELF['podman']['state'] = 'enabled';
+    SELF;
+};
```

Comment:
Or alternatively:
```suggestion
prefix '/software/components/systemd/unit/podman';
'startstop' = true;
'state' = 'enabled';
```

Reply:
The problem is that the service name is not the same on EL7 and EL8 so we cannot use the `prefix` construct...